### ISSUE TITLE
Staging Review finding: Downloadable PDF isn't screen reader friendly

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99005]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Representation Management
**Product:** Appoint a Representative MVP
**Feature:** Appoint a Representative + Representative Status Widget

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that meets best practices.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-meets-best-practices)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/appoint-rep/confirmation](https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/confirmation), you're prompted to download a PDF of your form, which you'll need to sign and then pass to your representative to also sign and submit.

It is very reasonable for a user to want to review their PDF before signing it to ensure that the information is correct. It's also very reasonable (I hope!) that the representative-to-be will also want to review the information in the PDF. But if you're a screen reader user (either the applicant or the representative), the PDF is not particularly easy to follow. The cover sheet doesn't announce any heading semantics, and the form itself announces sections in a seemingly random order.

It's worth noting that the Blinded Veterans Association is a VSO primarily staffed with people who are blind or low vision.

## Recommended action

I'm not an expert on PDF accessibility, but there are folks on the CAIA team who are, as are folks in the VA Section 508 Office. Reach out to them to ensure that the downloaded PDF will make sense to screen reader users.

If it's not possible to generate an accessible PDF version, then the next best option would be to generate an accessible version in another file type (HTML, txt, something else) that could be presented as an optional download.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by oddball-lindsay** [comment]
@opticbob @coforma-jamie and @EvanAtCoforma all met last week. The generated PDF's accessibility for screen readers has been improved and while the PDF still has room for improvement, the CAIA accessibility members said the current state is "on par" with other form experiences at VA. 

Closing this out for now and can pursue additional improvements to get "above par" after our MVP release.
